src/content/runtime-apis/request.md

```diff
@@ -39,8 +39,7 @@ The global `fetch` method itself invokes the `Request` constructor, thus the [`R
 
 <Aside header="Learn more">
 
-<!-- TODO(soon): /learning/understanding-the-request-context does not exist. -->
-Read [Understanding the FetchEvent Lifecycle](/learning/fetch-event-lifecycle) and [Understanding the Request Context](/learning/understanding-the-request-context) for a deeper understanding of these fundamental Workers concepts.
+Read [Understanding the FetchEvent Lifecycle](/learning/fetch-event-lifecycle) for a deeper understanding of these fundamental Workers concepts.
```

Comment:
We never migrated [The Request Context](https://web.archive.org/web/20200809045817/https://developers.cloudflare.com/workers/about/tips/request-context/) page from the prior docs.